### PR TITLE
Add create note pages

### LINKED ISSUE #3: Create new note UI

### PR DESCRIPTION
**Pull request by vivekmiyani**
Fixes #3.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -48,6 +50,23 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               rows={8}
               required
             />
+            <Select
+              className="flex-grow-0 w-full"
+              label="Assigned Contact"
+              name="contact"
+              options={Contacts}
+              placeholder="Select Contact"
+              required
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:59]: Do we need `required` here?

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 2]
```diff
@@ -48,6 +50,23 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
...
+            />
+            <Select
+              className="flex-grow-0 w-full"
+              isMulti
+              label="Tags"
+              name="tags"
+              options={Tags}
+              placeholder="Select Tags"
+              required
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:68]: Do we need required here?

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 3]
```diff
@@ -43,3 +43,25 @@ export const MenuItems = [
     },
   },
 ];
+
+export const Contacts = [
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/constants.js:47]: `CONTACTS_DATA`

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 4]
```diff
@@ -43,3 +43,25 @@ export const MenuItems = [
...
+    label: "Ronald Richards",
+    value: "some_uuid_1",
+  },
+  {
+    label: "Jacob Jones",
+    value: "some_uuid_2",
+  },
+];
+
+export const Tags = [
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/constants.js:58]: `TAGS_DATA`

File: app/javascript/src/constants/formInitialValues.js [hunk 5]
```diff
@@ -16,5 +16,7 @@ export default {
   notesForm: {
     title: "",
     description: "",
+    contact: null,
+    tags: null,
   },
 };
```
Comment (karthiknmenon) [inline app/javascript/src/constants/formInitialValues.js:22]: Please move the `initialValues` to a constants file in the scope of the `Notes` directory and name it as `NOTES_INITIAL_VALUES`.

File: app/javascript/src/constants/formValidationSchemas.js [hunk 6]
```diff
@@ -26,5 +26,7 @@ export default {
   notesForm: Yup.object().shape({
     title: Yup.string().required("Title is required"),
     description: Yup.string().required("Description is required"),
+    contact: Yup.object().required("Contact is required").nullable(),
+    tags: Yup.array().required("Tag is required").nullable(),
   }),
```
Comment (karthiknmenon) [inline app/javascript/src/constants/formValidationSchemas.js:31]: Please move the `validationSchema` to a constants file in the scope of the Notes directory and name it as `NOTES_VALIDATION_SCHEMA`.

**Review by karthiknmenon — COMMENTED** [review]
@vivekmiyani Please check the suggestions.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 1 above)
Comment (vivekmiyani) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:59]: I think yes as per the figma design.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 2 above)
Comment (vivekmiyani) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:68]: I think yes as per the figma design.

**Pull request merged**